Src/App/Router.php

```diff
@@ -3,27 +3,26 @@
 
 class Router
 {
-    private $interfaceControllerPath = '\Oat\Controller\ControllerInterface';
-
     /**
      * lead to the controller
+     * @param $dic intance of DIC class
      * @param $parameters
      * @return array
      * @throws \Exception
      */
-    public function route($parameters)
+    public function route($dic, $parameters)
```

Comment:
Why $dic is part of arguments and not before, it is now usefull to method logic?